patches/cli/0002-Exclude-test-projects-from-source-build.patch

```diff
@@ -36,5 +36,5 @@ index 406c4f8c1..62d043078 100644
 +  </PropertyGroup>
  </Project>
 -- 
-2.17.1.windows.2
+2.18.0
```

Comment:
Consider doing a pass to revert patch changes that don't actually change the patch effect?

Reply:
Good point, fixing these.